style/android/java/install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` , though it also looks like you could make this into a shell script by replacing `BASH_SOURCE[0]` with `$0` and replacing `[[` with `[`. If you do turn this into a shell script, `#!/bin/sh` is appropriate instead.